document_merge_service/api/tests/test_template.py

```diff
@@ -464,35 +465,82 @@ def test_template_merge_jinja_extensions_docx(db, client, template, settings, sn
     snapshot.assert_match(xml)
 
 
+@pytest.fixture
+def temp_png():
+    file_path = "/tmp/python-powered.png"
```

Comment:
Suggestion: use the `tmp_path` or `tmpdir` fixtures. They will also cleanup after themselves, so you don't have to

Reply:
Forgot about `tmp_path`, thanks! Done.